app/lib/settings_populator.rb

```diff
@@ -27,9 +27,14 @@ def initialize
 
   def populate
     ActiveRecord::Base.transaction do
+      puts "#{@defaults.size}"
+      updated = 0
+      inserted = 0
       @defaults.each do |hash|
         if Setting.exists?(key: hash['key'])
           Setting.find_by_key(hash['key']).update_attributes(hash.without('key', 'value'))
+          updated += 1
+          puts "#{updated}: #{hash['key']} updated" 
```

Comment:
We originally had puts statements here to show the changes being made to Settings but it was requested to remove them, especially for the test suite, because it cluttered up spec feedback. If we put a condition on them so that they don't run when running tests then I think that should be fine.

Reply:
Let me try to make that change and see if it pass travis

Reply:
oops, didn't see that you have already made that change.